syntax = "proto3";
package srv6_services;


enum StatusCode {
  STATUS_SUCCESS = 0;
  STATUS_OPERATION_NOT_SUPPORTED = 1;
  STATUS_BAD_REQUEST = 2;
  STATUS_INTERNAL_ERROR = 3;
  STATUS_INVALID_GRPC_REQUEST = 4;
  STATUS_FILE_EXISTS = 5;
  STATUS_NO_SUCH_PROCESS = 6;
  STATUS_INVALID_ACTION = 7;
  STATUS_GRPC_SERVICE_UNAVAILABLE = 8;
  STATUS_GRPC_UNAUTHORIZED = 9;
  STATUS_NOT_CONFIGURED = 10;
  STATUS_ALREADY_CONFIGURED = 11;
  STATUS_NO_SUCH_DEVICE = 12;
  INVALID_FWD_ENGINE = 13;
}
